wof/examples/production_configs/wsgi.py

```diff
@@ -1,6 +1,3 @@
 #! /home/ubuntu/miniconda/envs/wofpy/bin/python
-import sys
-sys.path.insert(0, "/var/www/timeseries")
```

Comment:
This doesn't seem to be needed since these setting will be in the same folder as `runserver.py`